lib/membrane_live_web/channels/event_channel.ex

```diff
@@ -81,6 +104,30 @@ defmodule MembraneLiveWeb.EventChannel do
     {:ok, Phoenix.Socket.assign(socket, %{peer_id: peer_id})}
   end
 
+  defp check_if_presenter(name, reloaded, id) do
+    [{_key, presenters}] = :ets.lookup(:presenters, id)
+    in_ets = if Enum.find(presenters, &(&1 == name)) != nil, do: true, else: false
```

Comment:
I think this if is useless here.
```suggestions
in_ets = Enum.find(presenters, &(&1 == name)) != nil
```